index.html

```diff
@@ -7,92 +7,226 @@
+            </div>
+            <div class="header_container flex-j-center flex">
+              <h3 class="paragraph_header">
+                Accesibility
+              </h3>
+            </div>
+            <div class="paragraph_description flex-j-center flex">
+              <p class="text">
+                Every Cabin is up in the nature but also easy to get to and to the towns around
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="pros_container">
+          <div class="photo_description_container">
+            <div class="pros_photo">
+              <img class="pros_pic" src="./mountains.png" alt="pic of pros">
+            </div>
+            <div class="header_container">
+              <h3 class="paragraph_header">Close to nature</h3>
+            </div>
+            <div class="paragraph_description flex-j-center flex">
+              <p class="text">Every cabin means different areas to explore!</p>
+            </div>
+          </div>
         </div>
       </div>
-    </div>
-    <div class="pros_container">
-      <div class="photo_description_container">
-        <div class="pros_photo"><img class="pros_pic" src="highway.png" alt="zdjęcie zalet"></div>
-        <div class="paragraph_container">
-          <header class="paragraph_header">Accesibility</header>
-          <p>Every Cabin is up in the nature but also easy to get to and to the towns around</p>
+    </section>
+    
+    <section class="pink">
+      <div class="flex section_wrapper">
+        <div class="left_side flex-j-center flex">
+          <div class="some_text">
+            <h2 class="third_page_header">
+              Work from somewhere special
+            </h2>
+            <p class="longer_paragraph">
+              We gather people who love nature and mountains and help them find perfect
+              spots
+              for a few weeks or even
+              months.
+              If your job is fully remote and you daydream of holidays, then why sit at home? Work as you
+              normally
+              do and
+              later let yourself dive into nature. Break the habit.
+            </p>
+          </div>
+          <div class="btn_third_sector">
+            <button class="book_now" type="submit">
+              <span class="book">Book Now</span>
+            </button>
+          </div>
+        </div>
+        <div class="right_side">
+          <img class="mountains_pic" src="./mountain_home.png" alt="mountain view">
         </div>
       </div>
-    </div>
-    <div class="pros_container">
-      <div class="photo_description_container">
-        <div class="pros_photo"><img class="pros_pic" src="mountains.png" alt="zdjęcie zalet"></div>
-        <div class="paragraph_container">
-          <header class="paragraph_header">Close to nature</header>
-          <p>Every cabin means different areas to explore!</p>
+    </section>
+
+
+    <section class=" fourth_page_container">
```

Comment:
Unnecessary space here, install:

Name: Trailing Spaces
Id: shardulm94.trailing-spaces
Description: Highlight trailing spaces and delete them in a flash!
Version: 0.3.1
Publisher: Shardul Mahadik
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces

Reply:
usage: ctrl + p, then type trailing spac... and choose correct option

Reply:
fixed, ctrl+p doesn't work it is working with F1 :)

Reply:
try also ctrl + shift + p

Reply:
works fine :)